ONSdigital/eq-author
# Fix formatting by running Prettier on all files

### What is the context of this PR?
- Fixes formatting differences that appear in PR's
- Prettier has been run on all files

### How to review 
- Run tests
- Run Prettier on all files and ensure no more files can be updated
